afs/templates/views/components/workflows/project-report-workflow/project-report-select.htm

```diff
@@ -12,4 +12,25 @@
             disabled: locked,
         }
     }"></div>
+    <!-- ko if: hasSetWithPhysicalThing() === false -->
+    <div class="tabbed-workflow-step-information-box widget-input-label" style="text-align: center; margin-top: 16px; height:fit-content; background-color: #fde9ad;">
+        This project has no associated Objects. Your project must have an associated Object Collection with an Object before proceeding.
+    </div>
+    <!-- /ko -->
+    <!-- ko if: termFilter() || physicalThingsValue() -->
+    <div style="padding: 5px; margin-top: 15px;">
+        <div class="widget-input-label">
+            <span>{% trans "Select an Object" %}</span>
```

Comment:
The title has updated.
But the question mark is a standard behavior of the resource-instance widget when widget can't get the iconclass for some unknown reason (possibly because the list is from term filter)